src/containers/my-resources/lessons-container/LessonsContainer.tsx

```diff
@@ -55,46 +54,47 @@ const LessonsContainer = () => {
     removeColumnRules
   )
 
-  const onResponseError = useCallback(
-    (error?: ErrorResponse) => {
-      dispatch(
-        openAlert({
-          severity: snackbarVariants.error,
-          message: getErrorKey(error)
-        })
-      )
-    },
-    [dispatch]
-  )
-
-  const getMyLessons = useCallback(
-    () =>
-      ResourceService.getUsersLessons({
-        limit: itemsPerPage,
-        skip: (page - 1) * itemsPerPage,
-        sort,
-        title: searchTitle.current,
-        categories: selectedItems
-      }),
-    [page, itemsPerPage, sort, searchTitle, selectedItems]
-  )
+  const getMyLessons = useCallback(async (): Promise<
+    ItemsWithCount<Lesson>
+  > => {
+    const response = await ResourceService.getUsersLessons({
+      limit: itemsPerPage,
+      skip: (page - 1) * itemsPerPage,
+      sort,
+      title: searchTitle.current,
+      categories: selectedItems
+    })
+    return response.data
+  }, [page, itemsPerPage, sort, searchTitle, selectedItems])
 
   const deleteLesson = useCallback(
     (id?: string) => ResourceService.deleteLesson(id ?? ''),
     []
   )
 
-  const { response, loading, fetchData } = useAxios<
-    ItemsWithCount<Lesson>,
-    GetResourcesParams
-  >({
-    service: getMyLessons,
-    defaultResponse: defaultResponses.itemsWithCount,
-    onResponseError
+  const {
+    data: response,
+    isLoading,
+    isError,
+    error,
+    refetch: fetchData
```

Comment:
Please, add meaningful name for that
```suggestion
    refetch: fetchLessons
```

Reply:
That's a good idea. Done.